### PR TITLE
i18n(fr): Update week 9

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
- [i18n(fr): Update preprcms.mdx](https://github.com/withastro/docs/commit/b167b600b8e164ff7fa821b258960b16c0b80b93)
- [i18n(fr): Update keystatic.mdx from](https://github.com/withastro/docs/commit/36bad637b8d6b883a20d77c2f53f8a6592ec675f) https://github.com/withastro/docs/commit/70f8b1158d1ea986d306e7ae8bb5d53b97bad9e8
- [i18n(fr): Update middleware.mdx from](https://github.com/withastro/docs/commit/fb8ed33e5cf1b36b3af7848c4573c56909bef029) https://github.com/withastro/docs/commit/42c181ce75356ff695c8e8a5c8280c2255d26e3b
- [i18n(fr): Update styling.mdx from](https://github.com/withastro/docs/commit/3862387e0010e64261bc6b84ab7c8eb6818fc7f1) https://github.com/withastro/docs/commit/b3adb293b8654234c353b78c9e386161ae194b21 
- [i18n(fr): Update nav.ts from](https://github.com/withastro/docs/commit/5137aa6bd1392cc0af4786db3ee3485db839f3f5) https://github.com/withastro/docs/commit/510c67d682ce2cfd5a3d12bb7c95bbaab84279d8 [(E-commerce)](https://github.com/withastro/docs/commit/5137aa6bd1392cc0af4786db3ee3485db839f3f5)
- [i18n(fr): Update ui.ts with feedback.createIssue.description](https://github.com/withastro/docs/pull/7109/commits/334c1fd55d41598f00a8d4d259583c1cab2bc732)

#### Related issues & labels (optional)

- Closes #
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/fr/guides/styling.mdx [hunk 1]
```diff
@@ -484,6 +484,27 @@ npm install less
...
+export default defineConfig({
+  vite: {
+    css: {
+      transformer: "lightningcss",
+    },
+  },
+})
+
+```
+
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/styling.mdx:507]: Can you add these changes recently [added](https://github.com/withastro/docs/commit/752278ad39b198fa40490d4c42acaf8d7d131976)

File: src/content/docs/fr/guides/cms/preprcms.mdx [hunk 2]
```diff
@@ -6,11 +6,227 @@ service: Prepr CMS
...
+    })
+    return response
+}
+```
+
+Votre répertoire racine doit maintenant contenir ces fichiers :
+
+<FileTree title="Project Structure">
+  - lib/
+  - **prepr.js**
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/cms/preprcms.mdx:60]:
```suggestion
    - **prepr.js**
```

File: src/content/docs/fr/guides/cms/preprcms.mdx [hunk 3]
```diff
@@ -6,11 +6,227 @@ service: Prepr CMS
...
+             _slug
+             title
+         }
+       }
+     }
+     `
+     export default GetArticles
+    ```
+
+3. Pour afficher une liste liée de vos articles de blog sur une page, importez et exécutez votre requête, y compris le point de terminaison Prepr nécessaire. Vous aurez alors accès à tous les titres de vos articles et à leurs slugs à rendre sur la page. (Dans l'étape suivante, vous allez [créer des pages individuelles pour vos articles de blog](#création-de-pages-individuelles-pour-les-articles-de-blog)).
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/cms/preprcms.mdx:92]:
```suggestion
3. Pour afficher une liste liée de vos articles de blog sur une page, importez et exécutez votre requête, y compris le point de terminaison Prepr nécessaire. Vous aurez alors accès à tous les titres de vos articles et à leurs slugs à rendre sur la page. (Dans l'étape suivante, vous allez [créer des pages individuelles pour vos articles de blog](#création-de-pages-individuelles-pour-les-articles-de-blog).)
```

File: src/content/docs/fr/guides/cms/preprcms.mdx [hunk 4]
```diff
@@ -6,11 +6,227 @@ service: Prepr CMS
...
+    ```
+
+
+Votre répertoire racine doit contenir ces nouveaux fichiers :
+
+<FileTree title="Project Structure">
+  - lib/
+  - prepr.js
+  - **queries**/
+  - **get-articles.js**
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/cms/preprcms.mdx:128]:
```suggestion
    - prepr.js
    - **queries**/
        - **get-articles.js**
```

File: src/content/docs/fr/guides/cms/preprcms.mdx [hunk 5]
```diff
@@ -6,11 +6,227 @@ service: Prepr CMS
...
+   ```
+
+Votre répertoire racine doit maintenant contenir ces nouveaux fichiers :
+
+<FileTree title="Structure du projet">
+  - lib/
+  - prepr.js
+  - queries/
+  - get-articles.js
+  - **get-article-by-slug.js**
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/cms/preprcms.mdx:213]:
```suggestion
    - prepr.js
    - queries/
        - get-articles.js
        - **get-article-by-slug.js**
```

File: src/content/docs/fr/guides/cms/preprcms.mdx [hunk 6]
```diff
@@ -6,11 +6,227 @@ service: Prepr CMS
...
+  - src/
+  - pages/
+  - index.astro
+  - **[…slug].astro**
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/cms/preprcms.mdx:217]:
```suggestion
    - pages/
      - index.astro
      - **[…slug].astro**
```

File: src/content/docs/fr/guides/cms/preprcms.mdx [hunk 7]
```diff
@@ -6,11 +6,227 @@ service: Prepr CMS
...
+
+Désormais, lorsque vous cliquez sur le lien d'un article dans la liste principale des articles de blog, vous accédez à une page contenant son contenu individuel.
+
+### Publier votre site
+
+Pour déployer votre blog Prepr, consultez nos [guides de déploiement](/fr/guides/deploy/) et suivez les instructions de votre hébergeur préféré.
 
 ## Ressources officielles
 
 - Consultez le [Prepr CMS Astro quickstart](https://github.com/preprio/astro-quick-start)
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/cms/preprcms.mdx:231]:
```suggestion
- Suivez le guide [Prepr CMS Astro quickstart](https://github.com/preprio/astro-quick-start) pour créer un blog simple avec Astro et Prepr CMS. 
```

File: src/content/docs/fr/guides/cms/preprcms.mdx [hunk 8]
```diff
@@ -6,11 +6,227 @@ service: Prepr CMS
...
-- Lire [Connecter Prepr CMS à Astro](https://docs.prepr.io/connecting-front-end-apps/astro)
\ No newline at end of file
+- Lire [Connecter Prepr CMS à Astro](https://docs.prepr.io/connecting-front-end-apps/astro)
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/cms/preprcms.mdx:232]:
```suggestion
- Consultez [Connecter Prepr CMS à Astro](https://docs.prepr.io/connecting-front-end-apps/astro) pour un aperçu des ressources Astro et Prepr CMS.
```

**Review by dreyfus92 — COMMENTED** [review]
Hop hop, le mises à jour rapides ⚡ Voici quelques suggestions 😁

File: src/content/docs/fr/guides/styling.mdx (hunk 1 above)
Comment (thomasbnt) [reply src/content/docs/fr/guides/styling.mdx:507]: Hey @dreyfus92, thanks for your notification about this recent edition. I commit that in https://github.com/withastro/docs/pull/7109/commits/c372ed104d6e636da940f6f75e8199692583efab

**Review by dreyfus92 — APPROVED** [review]
Merci beacoup @thomasbnt, LGTM ✅

**Comment by thomasbnt** [comment]
Merci à toi @dreyfus92 pour les reviews!

**Pull request merged**